### DISCUSSION TITLE
@StevenMapes is now a co-maintainer of this project

### DISCUSSION THREAD

**Discussion by twidi** [twidi/django-decorator-include#98] [Announcements]
Ok, I have to admit that this project was nearly dead. 

A long time ago, I added a first co-maintainer but like me, life took him away from this project.

I received numerous emails about the death of `django-decorator-include` and didn't answer them. My bad.

I don't know why, but this morning when receiving the nth notification about an issue being updated, I looked over it and my brain decided that it was the right time to talk and ask Steven to be a co-maintainer. He accepted.

So I guess it's the FOURTH birth of this project (after the original project creation, then me forking it, then the first co-maintainer, then Steven...)

 `django-decorator-include` really does not want to die :) 

Thanks to everyone for the issues and PRs, to those who tried to reach me, to the past maintainers, and now to Steven.

Long live `django-decorator-include`!

Twidi

**Comment by twidi** [comment]
Note: the project will stay under the same name & urls in Github and [Pipy](https://pypi.org/project/django-decorator-include/)

**Reply by kunambi** [reply]
This is excellent news! 

How long time does it typically take for pypi to publish the latest version?

**Reply by StevenMapes** [reply]
It won't be long I just had to pop out so I'm going to create the new main branch then I'll look to push the new version up

**Reply by StevenMapes** [reply]
> This is excellent news!
> 
> How long time does it typically take for pypi to publish the latest version?

3.1 is now up so you should be able to pull it down in the usual manner - https://pypi.org/project/django-decorator-include/

**Comment by twidi** [comment]
@StevenMapes I think it would be the perfect time to rename the `master` branch to `main` ;)

**Reply by StevenMapes** [reply]
Absolutely agree I'll get on that before the new release. I'm excited to take over the maintenance of the project and to keep this moving forward

**Reply by StevenMapes** [reply]
I've created a new branch called main but I do not have access to the repository settings so I can't update the default branch to main, if you could please do that @twidi then I'll start using that and develop can sit in the current state

**Reply by twidi** [reply]
it's done !

**Reply by StevenMapes** [reply]
Wonderful! Thank you so much

**Comment by StevenMapes** [comment]
Versions 3.1 and 3.2 have both been released. Both include the PRs to add in support for Django 4.0 through 5.1 as well as Python up to and including 3.14 so that should keep the package able to be installed in projects for a while. I'll add in tests and support for Django 5.2 once the alpha version is released in January

**Comment by twidi** [comment]
There is life again! 
![image](https://github.com/user-attachments/assets/eeb87bb7-1530-4361-9479-3a2afe337dda)

Thanks a lot @StevenMapes
